StringCalculatorKata/StringCalculatorKata.Tests/StringCalculatorShould.cs

```diff
@@ -0,0 +1,19 @@
+using Xunit;
+
+namespace StringCalculatorKata.Tests
+{
+    public class StringCalculatorShould
+    {
+        [Theory]
+        [InlineData("", 0)]
+        [InlineData("1", 1)]
+        [InlineData("1,2", 3)]
+        [InlineData("1\n2,3", 6)]
```

Comment:
what about if the string has non-integers? what's the result of `add`? it will throw an exception ;)

Reply:
and let's also think about short inputs, such as: "//", "//1", ...

Reply:
Okey I only followed your instruct "Make sure you only test for correct inputs. There is no need to test for invalid inputs for this kata."
but I'll handle them ;)
Should/could I use try catch for the exceptions in general?

Reply:
ah makes sense.

What kind of exceptions? Because some exceptions shouldn't be thrown and we need to fix the code to avoid them.

Reply:
Like if you try to parse a non-integer input.

Reply:
aha Tamam that's in the next req.

Reply:
nice work, thanks tamer!!
Can you add test cases for these cases:

"6,1,1,1000,,2000"
"10001,\n2000"
"10001,a,2000"
"10001,,2000"
","

Reply:
Thank you! 
I almost forgot about these general exceptions 😅
I'll add them of course ;)

Reply:
Done
you can review the pushed code

Reply:
Thanks Tamer! And general note, you need the code to handle the exceptions to NOT throw them, so if you pass a char not an integer, then you shouldn't parse it and just throw an error that 'the input is invalid'.
You should catch these exceptions from code and handle them, or the program will break.

Reply:
Done..
I handled any format exception other and make the code returns 0 in that case.
Does that work well?

Reply:
Fair enough :D